java/sage/FSManager.java

```diff
@@ -478,7 +492,12 @@ public void releaseISOMount(java.io.File mountDir)
         }
       }
       else
-        IOUtils.exec2(new String[] { "umount", mountDir.getAbsolutePath() }, false);
+      {
+        if (Sage.LINUX_IS_ROOT)
+          IOUtils.exec2(new String[]{"umount", mountDir.getAbsolutePath()}, false);
+        else
+          IOUtils.exec2(new String[]{"sudo","umount", mountDir.getAbsolutePath()}, false);
```

Comment:
Sorry, I saw the commit code before I saw the comments. I guess since root is the default configuration a user would actively be trying to run SageTV as a non-root user to see this kind of problem crop up. My emphasis was not so much that the user needs to be on the list so much as the command also need to be set to not require a password too, but both are important.